### PR TITLE
add faults to a guns that can have them

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
#74696 added some gun faults. Time to use them
#### Describe the solution
Simply add three unused `fault_fail_to_feed`, `fault_stovepipe`, `fault_double_feed` faults to gun that has `fault_gun_chamber_spent`
Also more details what Stovepipe is

### CODE REVIEW

File: data/json/items/gun/9mm.json [hunk 1]
```diff
@@ -754,7 +789,14 @@
       [ "stock mount", 1 ],
       [ "underbarrel mount", 1 ]
     ],
-    "faults": [ "fault_gun_blackpowder", "fault_gun_dirt", "fault_gun_chamber_spent" ],
+    "faults": [
+      "fault_gun_blackpowder",
+      "fault_gun_dirt",
+      "fault_gun_chamber_spent",
+      "fault_fail_to_feed",
+      "fault_stovepipe",
```
Comment (TheShadowFerret) [inline data/json/items/gun/9mm.json:797]:
```suggestion
```

File: data/json/items/gun/9mm.json [hunk 2]
```diff
@@ -717,7 +745,14 @@
       [ "sights mount", 1 ],
       [ "underbarrel mount", 1 ]
     ],
-    "faults": [ "fault_gun_blackpowder", "fault_gun_dirt", "fault_gun_chamber_spent" ],
+    "faults": [
+      "fault_gun_blackpowder",
+      "fault_gun_dirt",
+      "fault_gun_chamber_spent",
+      "fault_fail_to_feed",
+      "fault_stovepipe",
```
Comment (TheShadowFerret) [inline data/json/items/gun/9mm.json:753]:
```suggestion
```

**Review by TheShadowFerret — COMMENTED** [review]
No stovepipin' on your open bolters 😛

File: data/mods/Aftershock/items/gun/7.50mm.json [hunk 3]
```diff
@@ -175,7 +175,14 @@
       [ "sling", 1 ],
       [ "stock accessory", 2 ]
     ],
-    "faults": [ "fault_gun_blackpowder", "fault_gun_dirt", "fault_gun_chamber_spent" ],
+    "faults": [
+      "fault_gun_blackpowder",
+      "fault_gun_dirt",
+      "fault_gun_chamber_spent",
```
Comment (GuardianDll) [inline data/mods/Aftershock/items/gun/7.50mm.json:181]:
```suggestion
```

**Pull request merged**